tests/unit/test_processors.py

```diff
@@ -0,0 +1,229 @@
+"""Tests for Processor."""
+
+
+import json
+import os
+from pathlib import Path
+
+import pytest
+
+from circuit_maintenance_parser.processor import SimpleProcessor
+from circuit_maintenance_parser.data import NotificationData
+
+from circuit_maintenance_parser.parser import ICal
+from circuit_maintenance_parser.parsers.cogent import HtmlParserCogent1
+from circuit_maintenance_parser.parsers.gtt import HtmlParserGTT1
+from circuit_maintenance_parser.parsers.lumen import HtmlParserLumen1
+from circuit_maintenance_parser.parsers.megaport import HtmlParserMegaport1
+from circuit_maintenance_parser.parsers.telstra import HtmlParserTelstra1
+from circuit_maintenance_parser.parsers.verizon import HtmlParserVerizon1
+from circuit_maintenance_parser.parsers.zayo import HtmlParserZayo1
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+
+@pytest.mark.parametrize(
+    "processor_class, data_parsers, data_type, data_file, result_parse_file",
```

Comment:
Would it make more sense to pass in a list of `provider_class` and use each one's `processors` attribute in place of manually specifying combinations of `processor_class` and `data_parsers`? My concern here would be that this list could easily get out of sync with the actual provider/processor/parser combinations that we are actually implementing and caring about.

Alternately, is this test even needed, or is it functionally a duplicate of the testing in test_providers.py below?

Reply:
sorry @glennmatthews , this is completely outadated, I'm refactoring the tests completely